admin_manual/installation/apps_supported.rst

```diff
@@ -35,13 +35,16 @@ AGPL Apps
 Enterprise-Only Apps
 --------------------
 
+* Admin Audit
+* Collaborative tag management
 * Enterprise License Key
-* Files Drop
 * File Firewall
 * LDAP Home Connector
 * Log user and Sharing actions (1 new app, replacing the 2 former logging apps)
```

Comment:
The app is called 'Auditing' (in technical terms 'admin_audit' -> first on the list). You can also remove the hint in parentheses. I am not at all aware of two apps for this. Regular logging is in core and 'Auditing' is an Enterprise app.

Reply:
Thanks for the note. I'll take that out.